src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -1186,10 +1186,6 @@ def test_webapp_assign_system_identity(self, resource_group):
             self.cmd('webapp identity show -g {} -n {}'.format(resource_group, webapp_name), checks=[
                 self.check('principalId', result['principalId'])
             ])
-        self.cmd('role assignment list -g {} --assignee {}'.format(resource_group, result['principalId']), checks=[
```

Comment:
why remove this?

Reply:
Two reasons
1. the check did not seem to add any value to this specific test for MSI assignment for webapps
2. This call for role assignment list is returning an empty object -> even without my changes , ex. if i run this on PROD CLI  
![image](https://user-images.githubusercontent.com/13839372/55504881-af28f400-5606-11e9-8a7e-edc8b3634752.png)

I was trying not to take dependency on non webapp calls here - also honestly i don't understand the role assignment call & what the expected value should be here. If you believe this is valuable for the test i can create a bug to investigate & add this back, but i don't think this is blocking this PR. what do you think?

Reply:
We have a big regression here, if `role assignment list` returns nothing. Note, `'webapp identity assign` is supposed to create such a role assignment for you. Please do cross check here.

Reply:
Cross checked & if this is a regression i will start an investigation with the backend folks on this, but this is the behavior i am seeing on PROD as well. Can we keep this investigation orthogonal to the PR?

Reply:
// CC: @praries880 who is working on the Java SDK support on this. Please post update here.